src/printer.js

```diff
@@ -65,12 +64,14 @@ Printer.prototype.print = function(callback) {
 	async.eachSeries(
 		_self.commandQueue,
 		function(command, callback) {
-			if (_self.commandDelay !== 0) {
-				sleep.usleep(_self.commandDelay);
+			function write() {
+				_self.serialPort.write(command, function() {
+					_self.serialPort.drain(callback);
+				});
 			}
-			_self.serialPort.write(command, function() {
-				_self.serialPort.drain(callback);
-			});
+
+			if (_self.commandDelay === 0) return write()
```

Comment:
Yeah I felt the same about semicolons before I jumped on the bandwagon, but now I do feel that it looks really clean, and I haven't had one error because of it yet.

Re. tabs, it's really good in theory but I've always found it to yield more problems. For example right now when I copied the code you posted and pasted it into my editor, then indented it into the correct position. Everything looked good in the editor but two of the tabs where actually 2 spaces instead, I just didn't see it. Only noticed it because `git diff` uses 4 spaces for indentation...

Anyhow, each to his own :)